<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnPrincipalSM</class>
 <widget class="QMainWindow" name="vtnPrincipalSM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbBuscarCedula">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BUSCAR POR CEDULA: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBuscarCedula">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>BUSCAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbNombre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NOMBRE:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbApellido">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>APELLIDO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCedula">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CEDULA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbEdad">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>EDAD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSexo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SEXO:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellido">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCedula">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEdad">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cbSexo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SELECCIONAR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MASCULINO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FEMENINO</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnAgregar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>AGREGAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ACTUALIZAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbEspecialidad">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ESPECIALIDAD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbFecha">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FECHA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTotalPagar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TOTAL A PAGAR:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFecha">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbTipoUrgencia">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TIPO DE URGENCIA:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTotalPagar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbTipoUrgencia">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SELECCIONAR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BAJA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MEDIA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ALTA</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbHora">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HORA:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHora">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEspecialidad">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>25</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
